fuse/data/ops/ops_read.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         :param columns_to_extract: list of columns to extract from dataframe. When None (default) all columns are extracted
         :param rename_columns: rename columns from dataframe, when None (default) column names are kept
         :param key_name: name of value in sample_dict which will be used as the key/index
-        :param key_column: name of the column which use as key/index
+        :param key_column: name of the column which use as key/index. set to None to do nothing.
```

Comment:
In case of None, the original dataframe index will be used to extract the values for a single sample

Reply:
My bad. I don't even recall it 😅 

fixed.